File: conditionals/timeofday.c
// River Stanley, getting the time


#include <stdio.h>
#include <time.h>

int main(void){
    printf("Welcome, ");
    time_t seconds;
    seconds = time(NULL);
    time_t rawtime;
    struct tm * timeinfo;
    time(&rawtime);
    timeinfo = localtime(&rawtime);
    time_t now = time (NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;

if (hour<12){
    printf("good morning\n");
    
}else if (12<=hour<18){
    printf("good afternoon\n");
}else {
    printf("good evening\n");
}


return 0;
}

File: conditionals/timeofday.py
# river Stanley, time of day python 
import time
print("Welcome")
first_time= time.gmtime()
current= time.time()
now= time.ctime(current)
localtime = time.localtime(current)
day = localtime.tm_wday
hour = localtime.tm_hour 

if hour<12:
    print("good morning")
elif 12<=hour<18:
    print("good afternoon")
else:
    print("good evening")    